ocpp_codec/validators.py

```diff
@@ -216,3 +216,21 @@ def to_json(self, value):
                 value=value, enum_class=self.enum_class.__name__,
             )
         return value.value
+
+
+class OutgoingMessageDecimalEncoder(BaseEncoder):
+    """Encoder to limit the precision of decimal values sent to charge points.
+
+    This encoder doesn't restricts the precision of float values received from the charge point, but limits it to 6
```

Comment:
Maybe we should restricts for the received value too? The other side should not send us more than 6 places but it may not so sanitize data at entry point seems a good idea.

Reply:
In fact the specification is quite clear about this: `For data being reported by the Charging Station, the full resolution of the source data must be preserved.` That's why I'm not touching it, I'll add a comment.